triscord/__init__.py

```diff
@@ -92,6 +92,9 @@ def main(config_path, persist_path, debug=False):
         for action in feed_actions:
             message = feed.format_action(action)
             if message:
+                for trello_user, alias in settings.CONFIG.items('UserAlias'):
```

Comment:
This is a Trello message that we want to transform to be Discord-readable. As such let's do the alias substitution in the `trello` part of the program. Maybe in `TrelloActivityFeed(...).format_action()` ?

Reply:
Since the usernames we change are specific to Discord and are changed from Trello to Discord it makes sense in my opinion the the Discord module makes a message readable by Discord. This solution also makes Trello’s module more portable.

Reply:
The usernames we change are specific to Carpe Noctem, not Discord. TrelloActivityFeed.format_action merely formats an action to regular markdown.